content/doc/developer/plugin-development/choosing-jenkins-baseline.adoc

```diff
@@ -5,24 +5,37 @@ title: Choosing a Jenkins version to build against
 
 Your plugin's POM controls which version of Jenkins it is building/testing against.
 
-## Choosing a version
+== Choosing a version
 
 You need to balance compatibility and features:
 
-* *Keeping the Jenkins version your plugin builds against low* will allow more users to install and use your plugin. 
+* *Keeping the Jenkins version your plugin builds against low* will allow more users to install and use your plugin.
 In particular, the LTS Release Line is based on slightly older releases to provide a more stable experience for conservative users.
-* *Building against recent Jenkins versions* allows you to use recently added core features and API from your plugin. 
-You will also use that newer version for `mvn hpi:run`, so it may be easier to test your plugin with newer Jenkins releases. 
+* *Building against recent Jenkins versions* allows you to use recently added core features and API from your plugin.
+You will also use that newer version for `mvn hpi:run`, so it may be easier to test your plugin with newer Jenkins releases.
 Additionally, since features are sometimes moved from core into plugins, depending on a more recent Jenkins version will make your plugin's dependencies on what used to be core features explicit, otherwise your plugin will not work.
 * *Use at least the minimum version supported by the update center*, the supported update center versions can be found at link:https://updates.jenkins.io[updates.jenkins.io], at the time of writing (2020-03) this is 2.164.x.
```

Comment:
Update the example minimum version for Aug 2020?
```suggestion
* *Use at least the minimum version supported by the update center*, the supported update center versions can be found at link:https://updates.jenkins.io[updates.jenkins.io], at the time of writing (August 2020) this is 2.176.x.
```

Reply:
It’s recommend 2.204 elsewhere should it go to that instead? But this is fine too